### PR TITLE
Es label urls

### PR DESCRIPTION
**Pull request by hornmo**
- Fix ES label query
- Generalize Refdata Queries
- Add missing Refdata mappings

### CODE REVIEW

File: server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy [hunk 1]
```diff
@@ -318,33 +322,29 @@ class ESSearchService{
...
+  private void addRefdataToQuery(String value, QueryBuilder refdataQuery, String field){
     try{
-      status = RefdataValue.get(Long.valueOf(status))
+      value = RefdataValue.get(Long.valueOf(value))?.value
     }
     catch (Exception e){
     }
-    statusQuery.should(QueryBuilders.termQuery('status', status))
+    refdataQuery.should(QueryBuilders.matchQuery(field, value))
   }
```
Comment (VolkerHBZ) [inline server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy:348]: this is quite a wild construct - who invented it?

**Review by VolkerHBZ — APPROVED** [review]

File: server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy (hunk 1 above)
Comment (hornmo) [reply server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy:348]: Do you mean the whole method or the line?

Comment (VolkerHBZ) [reply server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy:348]: the whole method: parsing the string for numbers and ignoring the possible exception while using the (un-)altered value afterwards... Clearly, it's working, but looks... unusual and therefore weird.

Comment (hornmo) [reply server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy:348]: I agree it looks weird, but allows us to handle both id and value requests in a simple manner. I did not write this specific piece but I'm also fine with it as it is.

**Pull request merged**